src/openvpn/tmp-docker-entrypoint

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
```

Comment:
This is temp and only used for testing.  It is not called by the Dockerfile, but I left in case we need to go back and test in the future.  Alternatively I could remove this temp file and also the comments in the Dockerfile.